src/annotator/plugin/bucket-bar-js.js

```diff
@@ -0,0 +1,52 @@
+import { getBoundingClientRect } from '../highlighter';
+
+// FIXME: Temporary duplication of size constants between here and BucketBar
+const BUCKET_SIZE = 16; // Regular bucket size
+const BUCKET_NAV_SIZE = BUCKET_SIZE + 6; // Bucket plus arrow (up/down)
+const BUCKET_TOP_THRESHOLD = 115 + BUCKET_NAV_SIZE; // Toolbar
+
+/**
+ * Find the closest valid anchor in `anchors` that is offscreen in the direction
+ * indicated.
+ *
+ * @param {Object[]} anchors
+ * @param {'up'|'down'} direction
+ * @return {Object|null} - The closest anchor or `null` if no valid anchor found
+ */
+export function findClosestOffscreenAnchor(anchors, direction) {
+  const { next: closest } = anchors.reduce(
+    (acc, anchor) => {
```

Comment:
I know the name `acc` was taken from the existing code. I think `closest` might be a more obvious name for the accumulator in this context.

Reply:
Wow, I should have trusted my gut in so many ways yesterday :). I wanted to change the name, but felt like I was being too...change-y.